README.md

```diff
@@ -5,22 +5,31 @@
 
 # SDALib
 
-A framework to create AI for assisted cars for Linux and Windows. 
+A framework to create AI for assisted cars for Linux (64 bit) and Windows (32 bit) in C++ and Python.
 This Library can connect to the [DAISI](https://github.com/red-panda-productions/speed-dreams) simulation to adjust the controls of the human player in the simulation. The AI (currently) has full access to edit steer, acceleration and brake of the car.
```

Comment:
```suggestion
This Library can connect to the [DAISI](https://github.com/red-panda-productions/DAISI) simulation to adjust the controls of the human player in the simulation. The AI (currently) has full access to edit steer, acceleration and brake of the car.
```
- "The AI (currently) has full access to edit steer, acceleration and brake of the car." currently? doesn't it depend on the settings in DAISI?

Reply:
yes